tests/__init__.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
Can you elaborate on this file?

Reply:
Bah, forgot to pull the testdata func out from my other project. 

Basically, ln 1-14 setup the codecov env to dump coverage reports.